flake.nix

```diff
@@ -77,6 +77,9 @@
               SystemConfiguration
             ]);
 
+            copyBins = true;
+            copyDocsToSeparateOutput = true;
```

Comment:
I wonder, is it even useful to have docs in the Nix build output...? Couldn't someone just `cargo doc -p nix-installer --open` or visit `docs.rs/nix-installer`?

Reply:
Yes but we need to build the docs anyways for doctests xD

Reply:
...ah